.github/workflows/build-ncp-docker-images.yml

```diff
@@ -0,0 +1,44 @@
+name: Build NCP Docker Images
+
+on:
+  push:
+    branches: [ "main", "cfb/docker-8", "hbg/wave-8-nc"]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
We should remember to update this befor merging. I guess it should be simply
```
  push:
    branches: [ "main" ]
```
?

If we keep the `pull_request` part then there will be way too many images pushed to our ghcr.